integration/container/diff_test.go

```diff
@@ -12,22 +12,36 @@ import (
 )
 
 func TestDiff(t *testing.T) {
+	skip.If(t, testEnv.DaemonInfo.OSType == "windows", "cannot diff a running container on Windows")
+	ctx := setupTest(t)
+	apiClient := testEnv.APIClient()
+
+	cID := container.Run(ctx, t, apiClient, container.WithCmd("sh", "-c", `mkdir /foo; echo xyzzy > /foo/bar`))
+
+	expected := []containertypes.FilesystemChange{
+		{Kind: containertypes.ChangeAdd, Path: "/foo"},
+		{Kind: containertypes.ChangeAdd, Path: "/foo/bar"},
+	}
+
+	items, err := apiClient.ContainerDiff(ctx, cID)
+	assert.NilError(t, err)
+	assert.DeepEqual(t, expected, items)
+}
+
+func TestDiffStoppedContainer(t *testing.T) {
 	skip.If(t, testEnv.DaemonInfo.OSType == "windows", "FIXME")
 	ctx := setupTest(t)
 	apiClient := testEnv.APIClient()
 
 	cID := container.Run(ctx, t, apiClient, container.WithCmd("sh", "-c", `mkdir /foo; echo xyzzy > /foo/bar`))
 
-	// Wait for it to exit as cannot diff a running container on Windows, and
-	// it will take a few seconds to exit. Also there's no way in Windows to
-	// differentiate between an Add or a Modify, and all files are under
-	// a "Files/" prefix.
+	poll.WaitOn(t, container.IsInState(ctx, apiClient, cID, "exited"), poll.WithDelay(100*time.Millisecond), poll.WithTimeout(60*time.Second))
```

Comment:
Would this comment still be useful (as it gives some context)?

Is the timeout we use here the timeout we nowadays need for Windows? (I recall @vvoland created a `const` for it somewhere, but not sure if that's in this package)

Reply:
Yeah it's in the same package: `StopContainerWindowsPollTimeout`

Reply:
But, this test isn't supposed to execute on Windows, so I think we can just leave the default timeout here.

Reply:
> Would this comment still be useful (as it gives some context)?

It will yeah, I moved these tests a bit too much but yeah, the comment is useful

Reply:
I put the comment back now